docs/topics/native/native-ios-integration.md

```diff
@@ -55,6 +55,9 @@ If you want to call deinitialization on a special GC thread explicitly,
 set `kotlin.native.binary.objcDisposeOnMain=false` in your `gradle.properties`. This option
 enables deinitialization on a special GC thread, even if Swift/Objective-C objects were passed to Kotlin on the main thread.
 
+A special GC thread has to comply with the Objective-C runtime, meaning that it should have a run loop and
```

Comment:
Let's rephrase a little bit? Instead of `has to comply` it `complies`. The special thread is created by the K/N runtime and is not configurable by users in any way.

Reply:
Oh, also `it should have` should be `it has`, then, right?

Reply:
right, thanks for noticing!